README.md

```diff
@@ -0,0 +1,60 @@
+# iOS enhanced media player
+
+Repo for application in iOS for media player
+
+## Table of Contents
+
+- [Motivation](#motivation)
+- [Minimum deployment target](#minimum-deployment-target)
+- [Integration](#integration)
+- [Features](#features)
+- [Visual references](#visual-references)
+
+## Motivation
+
+A template project to build mobile iOS media player in Swift. 
```

Comment:
We aim to build a player like the comercial available ones (youtube, etc) using SwiftUI (modern) or something like that

Reply:
Don't explicitly use my words, just use the idea, but it needs to contain the idea that this player is used for training and practice the implementation of a media player using SwiftUI and modern practices